src/SFA.DAS.Tools.Support.Infrastructure/Services/EmployerCommitmentsService.cs

```diff
@@ -56,5 +60,33 @@ public async Task<StopApprenticeshipResult> StopApprenticeship(long employerAcco
                 };
             }
         }
+
+        public async Task<ApprenticeshipSummaryResult> GetApprenticeshipSummary(long apprenticeshipId, long employerAccountId)
+        {
+            try
+            {
+                if (employerAccountId <= 0)
+                {
+                    throw new ArgumentException("employerAccountId must be greater than 0", "employerAccountId");
+                }
+
+                if (apprenticeshipId <= 0)
+                {
+                    throw new ArgumentException("apprenticeshipId must be greater than 0", "apprenticeshipId");
+                }
+
+                var result = await _employerCommitmentApi.GetEmployerApprenticeship(employerAccountId, apprenticeshipId);
+
+                return _mapper.Map<ApprenticeshipSummaryResult>(result);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Failure to stop the apprenticeship.");
```

Comment:
`Failure to fetch apprenticeship` may be

Reply:
Have Updated Error Message